docs/reference/search/aggregations/bucket/datehistogram-aggregation.asciidoc

```diff
@@ -51,7 +51,12 @@ By default, times are stored as UTC milliseconds since the epoch. Thus, all comp
 done on UTC. It is possible to provide a time zone value, which will cause all computations to take the relevant zone
 into account. The time returned for each bucket/entry is milliseconds since the epoch of the provided time zone.
 
+ifdef::asciidoctor[]
+deprecated[1.5.0, "`pre_zone`, `post_zone` are replaced by `time_zone`"]
```

Comment:
I think it is worth adding the right number of `:`s here if we're doing `ifdef`.

Reply:
Fixed with 4ef1747

Reply:
Reverted with 0a524f5.

Adding a `:` to the `deprecated` macro actually changes the output. Gonna leave the `:` off so we mimic the current HTML.

Reply:
If you do `deprecated::[` it *should* work. the `::` gets you the block version, the `:` gets the inline version.

Reply:
Added with f2a2c329a8e90139a0cdbcc40d755a8ca2b4c23f.